src/result.js

```diff
@@ -171,7 +171,9 @@ class Result {
     body = HTMLSerializer.serializeChildren(body);
 
     this._head = head;
-    this._body = body;
+    
+    // Adding script boundary around the body
```

Comment:
Can you add a test for when `domContents` is called these script boundaries are always present?

Reply:
Thank you @kratiahuja, I added unit tests to verify domContents.body will always include boundary script tags.